README.md

```diff
@@ -1,3 +1,35 @@
 # REDCap to Harvest
 
 Dockerfile and tools for deriving Harvest applications from REDCap projects.
+
+Installation and Setup
+----------------------
+
+This tool runs using Docker. Install it if you don't have it:
+
+[Install Docker](http://docs.docker.com/installation/) 
+
+If you are using Mac, use boot2docker to run Docker.
```

Comment:
Mention of `boot2docker` is already in the installation guide you linked to, so this is redundant.